docs/src/index.md

```diff
@@ -26,3 +26,50 @@ notebooks in order to showcase this package's functionality.
 ```
 # put example notebooks here
 ```
+
+New to Julia
+-----------
+This documentation seeks to provide a higher-level description of the
+architecture and control-flow of the code, with references to the source code only
+to facilitate explanation. If the user seeks to understand the source code, while
+this documentation is a useful aid, aptitude and understanding of the Julia language
+(as well as object-oriented programming in general) will be necessary to fully
```

Comment:
FYI, it's debatable that Julia is object-oriented... I don't think there is universal agreement on that term, but usually "object-oriented" means you can do things like `object.myfunction()`. You can do that in Julia but its not "standard" (i.e. common style, likely to result in efficient code, etc.)